include/ametsuchi/pager/page.h

```diff
@@ -34,21 +40,50 @@ enum class SpanningItem : uint8_t {
   SINGLE_INFIX
 };
 
-
 struct PageDirectory {
   uint16_t offset;
   uint16_t length;
 };
 
+struct FileHeader {
+  uint8_t fmt_header[7];
+  uint8_t fmt_version;
+  uint8_t page_size_exponent;
+  uint16_t block_merkle_tree_width;
+  uint8_t block_merkle_tree_height;
+  uint16_t global_merkle_tree_width;
+  uint16_t hash_algo;
+};
 
-struct Page {
-  PageType pageType;
-  SpanningItem spanningItems;
-  uint16_t entryCount;
+constexpr FileHeader default_file_header = {
+    "天地",  //fmt_header[7]
```

Comment:
Replace UTF-8 literal constant with byte representation.
C++ does not guarantee UTF-8 source encoding.